fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionNyBedomningDiagnosgrund.java

```diff
@@ -0,0 +1,74 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.luae_na.v1.model.converter.certificate.question;
+
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.DIAGNOSGRUND_NYBEDOMNING_SVAR_DESCRIPTION_ID;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.DIAGNOSGRUND_NYBEDOMNING_SVAR_TEXT_ID;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.DIAGNOS_CATEGORY_ID;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.NYDIAGNOS_SVAR_ID_45;
+import static se.inera.intyg.common.support.facade.util.ValidationExpressionToolkit.singleExpression;
+
+import se.inera.intyg.common.services.texts.CertificateTextProvider;
+import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.support.facade.util.ValueToolkit;
+
+public class QuestionNyBedomningDiagnosgrund {
+
+    public static CertificateDataElement toCertificate(Boolean value, int index, CertificateTextProvider textProvider) {
+        return CertificateDataElement.builder()
+            .index(index)
+            .id(NYDIAGNOS_SVAR_ID_45)
+            .parent(DIAGNOS_CATEGORY_ID)
+            .config(
+                CertificateDataConfigRadioBoolean.builder()
+                    .id(DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45)
+                    .text(textProvider.get(DIAGNOSGRUND_NYBEDOMNING_SVAR_TEXT_ID))
+                    .description(textProvider.get(DIAGNOSGRUND_NYBEDOMNING_SVAR_DESCRIPTION_ID))
+                    .selectedText("Ja")
+                    .unselectedText("Nej")
+                    .build()
+            )
+            .value(
+                CertificateDataValueBoolean.builder()
+                    .id(DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45)
+                    .selected(value)
+                    .build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationMandatory.builder()
+                        .questionId(DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45)
```

Comment:
This should refer to NYDIAGNOS_SVAR_ID_45

Reply:
Fixed!